orchagent/mirrororch.cpp

```diff
@@ -1044,6 +1222,24 @@ void MirrorOrch::updateLagMember(const LagMemberUpdate& update)
         const auto& name = it->first;
         auto& session = it->second;
 
+        // Check the following conditions:
```

Comment:
if mirror session src port is port+lag,  then lag member update to null, the mirror session status will set to inactive ？